src/libs/SessionUtils.js

```diff
@@ -28,7 +29,27 @@ function isLoggingInAsNewUser(transitionURL, sessionEmail) {
     return linkedEmail !== sessionEmail;
 }
 
+/**
+ * Indicates whether the button should show the loading indicator
+ * @param {String} loadingScreen
+ * @param {Boolean} requiresTwoFactorAuth
+ * @param {Boolean} isLoading
+ * @returns {Boolean}
+ */
+function shouldShowLoadingIndicator(loadingScreen, requiresTwoFactorAuth, isLoading) {
```

Comment:
the `loadingScreen` param seems a bit out of place here, but I don't have any good suggestions for improvements. Maybe including a default or renaming `shouldShowLoadingIndicatorForScreen` could help make it clearer?